src/PositionManager.sol

```diff
@@ -136,7 +159,8 @@ contract PositionManager is
 
         // _beforeModify is not called here because the tokenId is newly minted
         BalanceDelta liquidityDelta = _modifyLiquidity(config, liquidity.toInt256(), bytes32(tokenId), hookData);
-
+        if (amount0Max < uint128(-liquidityDelta.amount0())) revert SlippageExceeded();
+        if (amount1Max < uint128(-liquidityDelta.amount1())) revert SlippageExceeded();
```

Comment:
why is this slippage check different from _increase? shouldn't mint and increase be the same?

Reply:
_mint is guaranteed to be negative delta so the conditional is cleaner

_increase can be positive (spill over fee revenue to be collected), where a slippage check is irrelevant